test/cpp/util/cli_credentials.cc

```diff
@@ -33,10 +33,15 @@
 
 #include "test/cpp/util/cli_credentials.h"
 
+#include <fstream>
+#include <sstream>
 #include <gflags/gflags.h>
 
 DEFINE_bool(enable_ssl, false, "Whether to use ssl/tls.");
 DEFINE_bool(use_auth, false, "Whether to create default google credentials.");
+DEFINE_string(ssl_server_ca_file, "", "Path to server CA file.");
```

Comment:
For these flags, please specify that --enable_ssl must be used and check that this is the case in the code.